src/cmap/test/integration.rs

```diff
@@ -58,3 +63,84 @@ async fn acquire_connection_and_send_command() {
     assert!(names.iter().any(|name| name == "admin"));
     assert!(names.iter().any(|name| name == "config"));
 }
+
+#[cfg_attr(feature = "tokio-runtime", tokio::test)]
+#[cfg_attr(feature = "async-std-runtime", async_std::test)]
+async fn concurrent_connections() {
+    let _guard = LOCK.run_exclusively().await;
+
+    let client = TestClient::new().await;
+    let version = VersionReq::parse(">= 4.2.9").unwrap();
+    // blockConnection failpoint option only added in 4.4.
```

Comment:
Based on this comment, shouldn't it be checking `>= 4.4.0`? 4.4 is already released, so we don't have to worry about 4.3.x or rc's anymore, and assuming the failpoint option isn't being backported to 4.2.x, this will fail if/when 4.2.10 comes out.

Reply:
Whoops, this comment is incorrect. It was added in 4.2.9 (see: https://jira.mongodb.org/browse/SERVER-41070). Updated the comment to match the version requirement.